### DISCUSSION TITLE
如何只代理dns,而实际访问走直连呢

### DISCUSSION THREAD

**Discussion by stryio** [v2fly/v2ray-core#911] [Q&A]
一些网站DNS被污染了，但是又不想用代理的ip访问他们

**Comment by lanchengjiu** [comment]
出口（outbounds）配置直连（freedom），但是设置（domainStrategy:UseIP），既使用V2Ray的DNS解析服务做出站解析（终结DNS）
至于哪些网站需要直连+本机DNS、哪些网站需要直连+V2RayDNS、哪些网站需要代理+V2RayDNS，就看你路由（routing）怎么配置了
